docs/codacy-guardrails/codacy-guardrails-getting-started.md

```diff
@@ -87,6 +92,33 @@ On the left side menu of the Codacy extension, please make sure that MCP server
 
 ### 4. Restart your IDE
 
+## How to install - Prerequisites for Windows (WSL)
+
+### 1. Ensure your machine has `node.js` and `npx` installed.
```

Comment:
not properly a windows wsl requirement, it's just an mcp requirement for all environments